Formula/afl-fuzz.rb

```diff
@@ -15,7 +15,7 @@ def install
     # behaving correctly!" in a nested login shell.
     # Reported to lcamtuf@coredump.cx 6th Apr 2016.
```

Comment:
Can you try it and see if it builds for you? If it does, we'll see if it builds on CI and if it does: we can remove the patch 😀